test/invariant/MainnetGovernorInvariants.t.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.19;
+
+import { IERC20 } from "oz/token/ERC20/IERC20.sol";
+import { IERC20Metadata } from "oz/token/ERC20/extensions/IERC20Metadata.sol";
+import "oz/utils/Strings.sol";
+import { Voter } from "./actors/Voter.t.sol";
+import { Proposer } from "./actors/Proposer.t.sol";
+import { BadVoter } from "./actors/BadVoter.t.sol";
+import { Fixture, AngleGovernor } from "../Fixture.t.sol";
+import { ProposalStore } from "./stores/ProposalStore.sol";
+
+//solhint-disable
+import { console } from "forge-std/console.sol";
+
+contract MainnetGovernorInvariants is Fixture {
+    uint256 internal constant _NUM_VOTER = 10;
+
+    Voter internal _voterHandler;
+    Proposer internal _proposerHandler;
+    BadVoter internal _badVoterHandler;
+
+    // Keep track of current proposals
+    ProposalStore internal _proposalStore;
+
+    function setUp() public virtual override {
+        super.setUp();
+
+        _proposalStore = new ProposalStore();
```

Comment:
I don't believe it will really be useful as it won't change anything to the actors having more delegation. In the end it is all just calling getVotes function which is already pretty well tested in fuzz and the other invariant